libs/zoekt/src/lib/zoekt.service.ts

```diff
@@ -0,0 +1,21 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { ZoektSearchRequest, ZoektSearchResponse } from './zoekt.declarations';
+
+const baseUrl = 'https://cors-anywhere.herokuapp.com/zoekt.achew22.com';
+
+@Injectable({ providedIn: 'root' })
+export class ZoektService {
+  search(zoektSearchRequest: ZoektSearchRequest) {
+    return this.http.post<ZoektSearchResponse>(
+      baseUrl + '/api/search',
+      zoektSearchRequest,
+      { headers: { 'Content-Type': 'application/json; charset=utf-8' } }
```

Comment:
I think this header is set automatically? Am I wrong

Reply:
It needs to have the charset=utf-8 at the end. The default doesn't transmit that and the rest API is really picky about it.